README.cn.md

```diff
@@ -0,0 +1,123 @@
+<p align="center">
```

Comment:
Add a link of README.cn.md in README.md?

Reply:
I want to replace README.md with a Chinese version since most of our users are in China.